src/components/TodoContainer.jsx

```diff
@@ -0,0 +1,151 @@
+    };
+
+    const axiosConfig = {
+        method: method,
+        url: `${AIRTABLE_URL}${url ?? ""}`,
+        headers: headers,
+        ...(body ? { data: body } : {})
+    };
+
+    try {
+        const response = await axios(axiosConfig);
+        return response.data;
+    } catch (error) {
+        throw new Error(`Error: ${error.response ? error.response.status : "Unknown"}`);
+    }
+};
+
+const savedSortOptions = JSON.parse(localStorage.getItem(LOCAL_STORAGE_SORT_KEY)) ?? {};
+const initialSort = savedSortOptions.sortBy ?? "lastModifiedTime";
+const initialIsReversed = savedSortOptions.isReversed ?? true;
+
+const TodoContainer = () => {
+    const [todoList, setTodoList] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [sortBy, setSortBy] = useState(initialSort);
+    const [isReversed, setIsReversed] = useState(initialIsReversed);
+
+    const addTodo = async (newTodo) => {
+        const airtableData = {
+            fields: {
+                title: newTodo
+            }
+        };
+        try {
+            await fetchAirtableData({ method: "POST", body: airtableData });
+        } catch (error) {
+            console.log(error.message);
+        }
+        await loadTodos();
+    };
+
+    const removeTodo = async (id) => {
+        try {
+            await fetchAirtableData({ method: "DELETE", url: `/${id}` });
+        } catch (error) {
+            console.log(error.message);
+        }
+        await loadTodos();
+    };
+
+    const loadTodos = async () => {
+        try {
+            const response = await fetchAirtableData({ method: "GET", url: `?${GRID_VIEW}` });
+
+            const todosFromAPI = await response;
+
+            setTodoList(
+                sortByField(
+                    todosFromAPI.records.map(todo => {
+                        return {
+                            id: todo.id,
+                            title: todo.fields.title,
+                            lastModifiedTime: todo.fields.lastModifiedTime
+                        };
+                    }),
+                        initialSort,
+                        initialIsReversed
+                ));
+        } catch (error) {
+            console.log(error.message);
+            setTodoList([]);
+        }
+    };
+
+    useEffect(() => {
+        (async () => {
+            setIsLoading(true);
+            await loadTodos();
+            setIsLoading(false);
+        })();
+    }, []);
+
+    useEffect(() => {
+        setTodoList(prevState => sortByField(prevState, sortBy, isReversed));
+    }, [isReversed, sortBy]);
+
+    const handleIsReversedChange = () => {
+        setIsReversed(prevState => {
+            const prevLocalStorage = JSON.parse(localStorage.getItem(LOCAL_STORAGE_SORT_KEY)) ?? {};
+            localStorage.setItem(LOCAL_STORAGE_SORT_KEY,
+                JSON.stringify({ ...prevLocalStorage, "isReversed": !prevState }));
+            return !prevState;
+        });
```

Comment:
Nitpick:
I'd recommend separating your SORT_BY and IS_REVERSED localStorage keys into separate fields and then move the logic into a single useEffect.

This will now drastically simplify your `handleIsReversedChange` (and `handleSortFieldChange`) callback function:
```js
const initialSort = localStorage.getItem(SORT_BY) ?? "lastModifiedTime";
const initiallsReversed = localStorage.getItem(IS_REVERSED) ?? true;

const App = () => {
  const [sortBy, setSortBy] = useState(initialSort);
  const [isReversed, setIsReversed] = useState(initiallsReversed);

  const handleIsReversedChange = () => {
    setIsReversed(prevState => !prevState);
  }

  useEffect(() => {
    localStorage.setItem(IS_REVERSED, isReversed);
    localStorage.setItem(SORT_BY, sortBy);
    setTodoList(prevState => sortByField(prevState, sortBy, isReversed));
  }, [isReversed, sortBy]);

  return ( ... );
}
```